docs/admin/admin-ui/scripts.md

```diff
@@ -0,0 +1,36 @@
+---
+tags:
+  - administration
+  - admin-ui
+  - admin
+  - role
+  - permission
+  - scripts
+  - mau
+---
+
+# Scripts
```

Comment:
What is the point of having two separate headings: Script and Custom Scripts? I think it's better to place the introduction directly under Custom Scripts.